audio-worklet/bitcrusher.html

```diff
@@ -42,33 +26,40 @@
   <head>
     <title>Bitcrusher</title>
     <link rel="stylesheet" href="style.css">
+    <script src="src/bitcrusher.js"></script>
+    <script src="src/sourcecontroller.js"></script>
+    <script src="src/paramcontroller.js"></script>
+    <script src="src/bitcrusher-demo.js"></script>
   </head>
 
   <body>
-    <div id="container" class = "container">
-      <div id="title"> Bit Crusher </div>
+    <h1>Bitcrusher: a lo-fi audio effect</h1>
+    
+    <h2>Hit play to start a song and listen to the distortive effect of 
+        modifying the bit depth and sample rate reduction parameters.
+    </h2>
+    
+    <h3>The bit depth slider controls the number of bits used to represent
+        a given sample. The sample rate reduction slider affects the number
+        of distinct samples in the signal. Increasing this value simulates
+        sample rate reduction by making intervals of consecutive samples equal.
+    </h3>
+    
+    <div id="container" class="container">
+      <h1 id="title">Bit Crusher</h1>
```

Comment:
The line 36 says `Bitcursher` but here the title is `Bit Crusher`. Make them consistent.